python_modules/dagster/dagster_tests/cli_tests/workspace_tests/test_grpc_server_workspace.py

```diff
@@ -147,6 +147,23 @@ def test_ssl_grpc_server_workspace(instance):
             pass
 
 
+def test_port_range(instance):
+    with environ({"DAGSTER_PORT_RANGE": "12345-12399"}):
```

Comment:
Kind of a brittle syntax. Are there other tools that use a similar pattern? Did you consider two env vars?

Perhaps not that big a deal as long as it's documented.